app/main.py

```diff
@@ -117,13 +141,15 @@ async def to_approve():
     return needs_approval
 
 
+
 @app.on_event("startup")
 @repeat_every(seconds=60 * 60 * 4)
 async def update():
     """ 1. scrape twitter for police use of force
         2. deduplicate data based on tweet id
         3. insert data into database
         4. repeat every 4 hours """
+# possible additions 'police', 'cop', 'policeman', 'cop', 'officer', 'cop', 'officers', 'officer', 'officers',
```

Comment:
Some of those keywords, although seemingly helpful (I thought they were too), will end up being too general and may cause highly skewed results with the level 0 data.  The search has been updated though to include a much larger range including some of the terms that are listed here.  Please comment out before final PR!

Reply:
Good catch that is something that I need to take out.